.github/workflows/rotki_nightly_builds.yml

```diff
@@ -61,6 +61,16 @@ jobs:
           path: |
             ${{ steps.packaging.outputs.binary }}
             ${{ steps.packaging.outputs.binary_checksum }}
+      - name: Notify On Failure
+        if: ${{ failure() }}
+        run: |
+          curl --request POST \
+            --url ${{ secrets.WEBHOOK }} \
+            --header 'Content-Type: application/json' \
+            --data '{
+          	"content": ":construction_site: **Github Actions:** ${{ github.job }} failed :x: \r\n\r\n  <@&735068019440615516> please have a look at [${{ github.run_id }}](https://github.com/rotki/rotki/actions/runs/${{ github.run_id }}) :cry:"
```

Comment:
Lol at the :cry: emoji

Reply:
Well if the build fails 😢  is the proper reaction 😄